components/tidb_query_expr/src/lib.rs

```diff
@@ -599,6 +619,7 @@ fn map_expr_node_to_rpn_func(expr: &Expr) -> Result<RpnFnMeta> {
         ScalarFuncSig::PeriodDiff => period_diff_fn_meta(),
         ScalarFuncSig::LastDay => last_day_fn_meta(),
         ScalarFuncSig::AddDurationAndDuration => add_duration_and_duration_fn_meta(),
+        ScalarFuncSig::MakeTime => map_make_time_func(value, children)?,
```

Comment:
I think the first argument is always signed

Reply:
OK，but this func in TiDB has judged it

Reply:
> I think the first argument is always signed

OK，but this func in TiDB has judged it

Reply:
Could you tell me which line?

Reply:
> Could you tell me which line?

[https://github.com/pingcap/tidb/blob/master/expression/builtin_time.go#L5838](url)
`	dur, err = b.makeTime(hour, minute, second, mysql.HasUnsignedFlag(b.args[0].GetType().Flag))
`

Reply:
Interesting~ Anyway, you can capture `args`, which has type  `&[crate::RpnStackNode<'_>]`, so only one rpn fn is sufficient.

Reply:
> Interesting~ Anyway, you can capture `args`, which has type `&[crate::RpnStackNode<'_>]`, so only one rpn fn is sufficient.

I see...maybe I will push new code after get off the work.